tests/scene/test_text_edit.h

```diff
@@ -7355,9 +7355,9 @@ TEST_CASE("[SceneTree][TextEdit] multicaret") {
 		CHECK(text_edit->get_caret_line(1) == 1);
 		CHECK(text_edit->get_caret_column(1) == 1);
 		CHECK(text_edit->get_caret_line(2) == 0);
-		CHECK(text_edit->get_caret_column(2) == 8);
+		CHECK(text_edit->get_caret_column(2) == 7);
 		CHECK(text_edit->get_caret_line(3) == 0);
-		CHECK(text_edit->get_caret_column(3) == 4);
+		CHECK(text_edit->get_caret_column(3) == 2);
```

Comment:
Note: this test uses default non-monospaced font (there's no exact above position) and extremely small width (which make it pretty useless to test real control usage), so numbers are pretty arbitrary. New values look reasonable if test conditions recreated manually:
<img width="25" alt="Screenshot 2025-03-19 at 12 56 14" src="https://github.com/user-attachments/assets/6b5ce8b1-9047-45e0-a6ca-96109edccb54" />